packages/core/src/index.ts

```diff
@@ -17,7 +17,7 @@
  */
 
 export * from './api/public-api';
-export {default as getBranding} from './branding/branding';
+export * from './branding/public';
```

Comment:
It's not strictly necessary to use an `index.ts` file. But as the main entry point for the package, wouldn't it be good to follow the common convention? 
This could improve the readability of the code as well.